certificate-service/domain/src/main/java/se/inera/intyg/certificateservice/domain/certificate/model/Certificate.java

```diff
@@ -68,6 +70,7 @@ public void updateMetadata(ActionEvaluation actionEvaluation) {
         .careUnit(actionEvaluation.careUnit())
         .careProvider(actionEvaluation.careProvider())
         .issuingUnit(actionEvaluation.subUnit())
+        .externalReference(actionEvaluation.externalReference())
```

Comment:
Not sure that externalReference should be updated if already provided. Do you know? Best to double check with Webcert and current implementation.